packages/gatsby/src/utils/webpack.config.js

```diff
@@ -434,6 +434,49 @@ module.exports = async (
     }
   }
 
+  if (stage === `build-html` || stage === `develop-html`) {
+    const externalList = [
+      /^lodash/,
+      `react`,
+      /^react-dom/,
```

Comment:
Should this have `lodash` and `react-dom` as strings along with the regexes?

Reply:
oh wait. ignore me 🤦‍♂️

Reply:
`lodash` is regexp because otherwise it wouldn't catch things like `require(`lodash/merge`)

`react-dom` could be changed to string if I would use `"react-dom"` and `"react-dom/server"`

Reply:
When I use this with redux, 'develop' fails because of lodash-es package